libadaptivity/load_balance/src/Mesh.cpp

```diff
@@ -779,53 +779,53 @@ samfloat_t Mesh::idealElementDensity(const unsigned eid)
     }
     unsigned len = nodes.size();
     a1 /= len; a2 /= len; b1 /= len; b2 /= len;
-    samfloat_t det = a1*b2 - a2*b1;
-    return sqrt(det)*elementVolume(eid);
+    det = a1*b2 - a2*b1;
   }
+  return sqrt(det)*elementVolume(eid);
```

Comment:
If neither branch is taken above, this will still be using the uninitialised value of `det`.